core/settings.py

```diff
@@ -161,6 +161,8 @@ def get_env_variable(name, cast=str, default=""):
     SHELL_PLUS_PRINT_SQL = get_env_variable(
         "SHELL_PLUS_PRINT_SQL", default=True, cast=bool
     )
+    SHELL_PLUS = get_env_variable("SHELL_PLUS", default="ptpython")
```

Comment:
I have a personal preference for `ipython`, this change allows me to use it with a change in my local `.env`